commit ea4e8d8adac0f64cebf9279451c2e7abd570ba7e
Author: David <DavidCrompton@duck.com>
Date:   Sun Jan 12 19:26:32 2025 +0000

    IMG-ROGUE Fix

diff --git a/drivers/gpu/drm/img/img-rogue/Kconfig b/drivers/gpu/drm/img/img-rogue/Kconfig
index 5b793bb9d..6f84a2643 100644
--- a/drivers/gpu/drm/img/img-rogue/Kconfig
+++ b/drivers/gpu/drm/img/img-rogue/Kconfig
@@ -1,5 +1,5 @@
 config DRM_IMG_ROGUE
-	bool "DRM support for PowerVR GPU"
+	tristate "DRM support for PowerVR GPU"
 	select DRM_IMG
 	default n
 	help
diff --git a/drivers/gpu/drm/img/img-rogue/Makefile b/drivers/gpu/drm/img/img-rogue/Makefile
index ee37ef8ca..19c1d8ffb 100644
--- a/drivers/gpu/drm/img/img-rogue/Makefile
+++ b/drivers/gpu/drm/img/img-rogue/Makefile
@@ -73,7 +73,7 @@ WINDOW_SYSTEM=nulldrmws
 #include $(OUT)/config_kernel.mk
 include $(srctree)/$(src)/config_kernel.mk
 
-.SECONDARY:
+.NONINTERMEDIATE:
 
 define symlink-source-file
 @if [ ! -e $(dir $@) ]; then mkdir -p $(dir $@); fi
diff --git a/drivers/gpu/drm/img/img-rogue/services/server/devices/rogue/rgxdebug.c b/drivers/gpu/drm/img/img-rogue/services/server/devices/rogue/rgxdebug.c
index bccced1bf..8653ac774 100644
--- a/drivers/gpu/drm/img/img-rogue/services/server/devices/rogue/rgxdebug.c
+++ b/drivers/gpu/drm/img/img-rogue/services/server/devices/rogue/rgxdebug.c
@@ -2375,7 +2375,7 @@ static void _FillAppForFWFaults(PVRSRV_RGXDEV_INFO *psDevInfo,
 			{
 				OSStringLCopy(psProcInfo->szProcessName,
 								"(unknown)",
-								sizeof(psProcInfo->szProcessName));
+								16);//sizeof(psProcInfo->szProcessName));
 			}
 		}
 	}
diff --git a/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk b/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
index f3bf4f1cc..b8faefc4b 100644
--- a/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
+++ b/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
@@ -42,6 +42,8 @@
 # Window system
 ccflags-y += -DWINDOW_SYSTEM=\"$(WINDOW_SYSTEM)\"
 
+ccflags-y += -D__NO_FORTIFY
+
 # Linux kernel headers
 ccflags-y += \
  -Iinclude \
@@ -163,6 +165,7 @@ else
 endif
 
 ifeq ($(SUPPORT_RGX),1)
+CFLAGS_rgxdebug.o := -D__NO_FORTIFY
 $(PVRSRV_MODNAME)-y += \
  services/server/devices/rgx_bridge_init.o \
  services/server/devices/rgxfwdbg.o \
