diff --git a/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk b/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
index f3bf4f1cc..b8faefc4b 100644
--- a/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
+++ b/drivers/gpu/drm/img/img-rogue/services/server/env/linux/Kbuild.mk
@@ -42,6 +42,8 @@
 # Window system
 ccflags-y += -DWINDOW_SYSTEM=\"$(WINDOW_SYSTEM)\"
 
+ccflags-y += -D__NO_FORTIFY
+
 # Linux kernel headers
 ccflags-y += \
  -Iinclude \
@@ -163,6 +165,7 @@ else
 endif
 
 ifeq ($(SUPPORT_RGX),1)
+CFLAGS_rgxdebug.o := -D__NO_FORTIFY
 $(PVRSRV_MODNAME)-y += \
  services/server/devices/rgx_bridge_init.o \
  services/server/devices/rgxfwdbg.o \
